src/firefox/index.ts

```diff
@@ -130,7 +130,7 @@ export default class {
     })
 
   async launch() {
-    const isRunning = await find('name', 'Firefox')
+    const isRunning = await find('name', /firefox*/gi)
```

Comment:
This is not ideal and ideally we should have a different name for our `point-browser`. This code will fail to launch our "Firefox" point-browser if user already has a normal Firefox running (which is not point-browser) since the regex (regex not being the culprit here, the culprit being not having unique name) will match and thus preventing our "Firefox" point-browser launch.

Reply:
Yes. We need a way to change the name of the process.

Reply:
I had to comment that piece of code in the latest commits because it was preventing Firefox from launching. In Windows task manager, I noticed 4 processes which were launched before applying the fix. After the fix was applied, I believe it must be only allowing 1 process to launch and blocking the others which explains why (for me) Firefox wasn't launching anymore.